scripts/dev/dev_config.py

```diff
@@ -107,6 +107,14 @@ def test_data_dir(self) -> str:
     def readiness_probe_image_dev(self) -> str:
         return self._get_dev_image("readiness_probe_image_dev", "readiness_probe_image")
 
+    @property
+    def mongodb_image_name(self) -> str:
```

Comment:
with this we don't panic for current users. But it is not optimal as we, again, split up configurations across the place. 

In example: `test/e2e/setup/test_config.go`, here `scripts/ci/config.json` etc.


I am open in importing one of these here and use those defaults instead.